validator/sawtooth_validator/execution/context_manager.py

```diff
@@ -224,8 +224,13 @@ def commit_context(self, context_id_list, virtual):
             merged_updates.update(context.get_writable_address_value_dict())
 
         new_root = merkle_root
-        add_value_dict = {address: value.result()
-                          for address, value in merged_updates.items()}
```

Comment:
is this better remaining as a comprehension:
add_value_dict = {address: value.result()
                          for address, value in merged_updates.items() if value.result() is not None}